puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/scm/foreman-discovery-image.yml

```diff
@@ -0,0 +1,9 @@
+- scm:
+  name: foreman-discovery-image
```

Comment:
I think it might complain about this needing to be indented 2 more spaces.

Reply:
Oh okay, I thought since it was indented from the `-`, it'd parse properly. I can update.

Reply:
I was incorrect! I tried it on http://yaml-online-parser.appspot.com/. they parsed differently.